tests/ui/parser/shebang/shebang-doc-comment.stderr

```diff
@@ -2,9 +2,12 @@ error: expected item, found `[`
   --> $DIR/shebang-doc-comment.rs:2:1
    |
 LL | [allow(unused_variables)]
-   | ^ expected item
+   | ^------------------------
+   | |
+   | unexpected expression
    |
-   = note: for a full list of items that can appear in modules, see <https://doc.rust-lang.org/reference/items.html>
+   = help: consider putting it inside a function: fn main() { [allow(unused_variables)] }
```

Comment:
Hm, I don't think we want to have this recovery, at least when you could parse it like a macro attr w/o a `#`.

Reply:
Hm it thinks that it's an expression so it suggests putting it inside a function. 
How can we tell that this is a bad attr and not an array literal with a function call?